ci/jenkins/test.sh

```diff
@@ -345,12 +345,16 @@ function prepare_env {
 function revert_snapshot_windows {
     WIN_NAME=$1
     echo "==== Reverting Windows VM ${WIN_NAME} ====="
-    if [[ $WIN_NAME == *"jumper"* ]]; then
-        govc snapshot.revert -vm ${WIN_NAME} win-initial
+    if [[ ${TESTCASE} =~ "containerd" ]]; then
+        if [[ $WIN_NAME == *"jumper"* || $WIN_NAME == *"win-image"* || ${TESTCASE} == *"userspace"* ]]; then
```

Comment:
Are you sure `userspace` case is valid here? Currently there is no "containerd" in the e2e_type of the userspace job.